docs/_custom-rule-tutorial-code/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+module.exports = {
```

Comment:
I think we should use flat config format for this example.

Reply:
updated the tutorial to use this @nzakas!